engine/beef_bump.go

```diff
@@ -31,7 +30,7 @@ func calculateMergedBUMP(txs []*Transaction) (BUMPs, error) {
 	for _, k := range mapKeys {
 		bump, err := CalculateMergedBUMP(bumps[k])
 		if err != nil {
-			return nil, fmt.Errorf("Error while calculating Merged BUMP: %s", err.Error())
+			return nil, spverrors.Wrapf(err, "Error while calculating Merged BUMP")
```

Comment:
My bad in previous function, but I think error should be lowercased 🙈

Reply:
you're probably right,
I must admit that I simply do a find&replace so I didn't check previous messages.
I can fix this one, but I'm not sure if there are more of them.

Reply:
Done